apiserver/src/api.rs

```diff
@@ -0,0 +1,230 @@
+use crate::error::{self, Result};
+use models::node::{BottlerocketNodeClient, BottlerocketNodeSelector, BottlerocketNodeStatus};
+
+use actix_web::{
+    middleware,
+    web::{self, Data},
+    App, HttpRequest, HttpResponse, HttpServer, Responder,
+};
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+use snafu::ResultExt;
+
+const NODE_RESOURCE_ENDPOINT: &'static str = "/bottlerocket-node-resource";
+
+/// HTTP endpoint that implements a shallow health check for the HTTP service.
+async fn health_check() -> impl Responder {
+    HttpResponse::Ok().body("pong")
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+/// Describes a node for which a BottlerocketNode custom resource should be constructed.
+pub struct CreateBottlerocketNodeRequest {
+    pub node_selector: BottlerocketNodeSelector,
+}
+
+/// HTTP endpoint which creates BottlerocketNode custom resources on behalf of the caller.
+async fn create_bottlerocket_node_resource<T: BottlerocketNodeClient>(
+    _req: HttpRequest,
+    settings: web::Data<APIServerSettings<T>>,
+    create_request: web::Json<CreateBottlerocketNodeRequest>,
+) -> Result<impl Responder> {
+    let br_node = settings
```

Comment:
Maybe add some `trace!` logging here, and log some information from the http request? Or are we getting logs from actix?

Reply:
I've been investigating adding support for tracing via [tracing-actix-web](https://github.com/LukeMathWalker/tracing-actix-web); however, the changes are large enough that I'd like to add them in a subsequent PR. I've opened #94 to track that. Is it okay to proceed here without that feature for now?